### PR TITLE
Add NY datathon webpage (publish Sept 16)

### PR DESCRIPTION
**Pull request by SamFritz**
Addresses [Add Webpage for fourth datathon #134](https://github.com/archivesunleashed/archivesunleashed.org/issues/134)

This PR incorporates the following information onto the webpage: 

-  Intro image (save the date)
-  Information about the datathon
-   Organizers
-   Sponsor logos
-   Important Dates
-   Submission details
-   Travel grants

Change from previous datathon pages: sponsor logos were created/displays as a single .png rather then in a table format which alters logo size too much. 

**Test/Review**
 @ianmilligan1 @ruebot would appreciate eyes on this page just to make sure we haven't left out any important details, graphics/visuals display nicely, etc. There is no rush, just want to prep the PR. Always happy for any style/content suggestions!
_Edits completed on ny-datathon branch_

**Note**: Let's publish on Sept 16, 2019 pls. Don't want webpage to be visible before save the date goes out on Sept. 9. Thanks!!

### CODE REVIEW

File: content/new-york/index.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+The project team invites archivists, researchers (from the humanities, social sciences, and beyond), librarians, computer scientists, and web archiving enthusiasts to join us over the course of two days to collaboratively work with web collections and explore cutting-edge research tools through hands-on experience.
+
+The Archives Unleashed Team has partnered with <a href="https://library.columbia.edu">**Columbia University Libraries**</a> to host the fourth Archives Unleashed datathon.
+
+This event will bring together a small group of **approximately 15-20 participants** to experiment with the newest release of AUT (Archives Unleashed Toolkit) and the Archives Unleashed Cloud, and to kick-off collaboratively inspired research projects. Participants will have access to analytics software and specialists, and will be exposed to the process of working with web archive files at scale.
+
+For more information on AUT and the Cloud, please visit http://archivesunleashed.org/.
```
Comment (ianmilligan1) [inline content/new-york/index.md]: AUT and the Cloud ->

"Archives Unleashed Toolkit and Cloud"

File: content/new-york/index.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
+---
+title: New York Datathon
+date: 2017-12-05T15:20:57-05:00
+weight: 45
+aliases:
+  - "New York"
+---
+
+![New York Skyline](/images/ny-skyline.png)
```
Comment (ianmilligan1) [inline content/new-york/index.md]: If we decide to change the images to feature Columbia's library a bit more, let's update this too

Comment (SamFritz) [reply content/new-york/index.md]: oh yes, the one place I forget to change

**Comment by SamFritz** [comment]
Let's Merge this PR on Monday when the CFP goes live :)

**Pull request merged**